src/main/java/core/basesyntax/RectangleSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class RectangleSupplier {
+    private static final RandomGeometricValue randomSize = new RandomGeometricValue();

```

Comment:
The instance variables `randomSize` and `color` should not be static since they don't represent a single shared value for all instances of the `RectangleSupplier` class.